tweak: Nanocalcium fixes bones and bleedings faster

Ksaikok46 opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
Нанокальций стал чинить сломанные кости и внутренние кровотечения в 2 раза быстрее, количество вещества в шприце также уменьшилось в 2 раза.
**Было:**
Шприц вкалывал 30 юнитов нанокальция, которые полностью усваиваются за 60 циклов, лечение переломов/внутряков начиналось на 45 цикле с шансом 25% в цикл.
**Стало:**
Шприц вкалывает 15 юнитов нанокальция, которые полностью усваиваются за 30 циклов, лечение переломов/внутряков начинается на 23 цикле с шансом 50% в цикл. 
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Ссылка на предложение/Причина создания ПР
https://discord.com/channels/617003227182792704/755125334097133628/1231376084096778301
<!-- Здесь оставьте ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что ваше предложение одобрено. Либо укажите, почему этот ПР должен пройти без предложки. -->
<!-- Пример ссылки: https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

Ksaikok46 committed:
speed up

Gottfrei commented on code/modules/reagents/chemistry/reagents/medicine.dm:
```diff
@@ -1373,27 +1373,27 @@
 				if(ishuman(M))
 					var/mob/living/carbon/human/H = M
-					for(var/obj/item/organ/internal/I as anything in M.internal_organs) // 60 healing to all internal organs.
+					for(var/obj/item/organ/internal/I as anything in M.internal_organs) // 28 healing to all internal organs.
 						I.heal_internal_damage(4)
```
а почему лечение и урон ниже не проскалировал?

Ksaikok46 replied:
> а почему лечение и урон ниже не проскалировал?

посовещался с автором предложки, но если думаешь что лучше проскалировать, то могу сделать
![image](https://github.com/ss220-space/Paradise/assets/139886553/f2b6a7c4-3ae3-47e5-aa2d-801bd68f6c68)

Gottfrei replied:
просто я не вижу никаких упоминаний в ТЗ. раз мы всё скалируем, то пускай будет всё.

Ksaikok46 committed:
damage/heal scale

все значения увеличены в два раза, так как усваивается нанокальций в два раза быстрее, значения спутанности/тряски особо не вижу смысла менять

Daeberdir commented:
Зажлобил 1 лишний цикл, жесть 😿

Daeberdir commented on code/modules/reagents/chemistry/reagents/medicine.dm:
```diff
@@ -1356,55 +1356,55 @@
+					M.reagents.remove_reagent(R.id, 1) //We will be generous (for nukies really) and purge out the chemicals during this phase, so they don't fucking die during the next phase. Of course, if they try to use adrenals in the next phase, well...
+		if(10 to 21)
 			//If they have stimulants or stimulant drugs then just apply toxin damage instead.
 			if(has_stimulant == TRUE)
```
```suggestion
			if(has_stimulant)
```
Небольшое прихорашивание.

Daeberdir commented on code/modules/reagents/chemistry/reagents/medicine.dm:
```diff
@@ -1356,55 +1356,55 @@
+			to_chat(M, span_warning("Your body goes rigid, you cannot move at all!"))
+			M.AdjustWeakened(15 SECONDS)
+		if(23 to INFINITY) // Start fixing bones | If they have stimulants or stimulant drugs in their system then the nanites won't work.
 			if(has_stimulant == TRUE)
```
```suggestion
			if(has_stimulant)
```

Daeberdir commented on code/modules/reagents/chemistry/reagents/medicine.dm:
```diff
@@ -1345,8 +1345,8 @@
 	harmless = FALSE
-	taste_description = "2 minutes of suffering"
-	var/list/stimulant_list = list("methamphetamine", "crank", "bath_salts", "stimulative_agent", "stimulants")
+	taste_description = "1 minute of suffering"
```
```suggestion
	taste_description = "minute of suffering"
```
Вот тут не уверен, как ему лучше выглядеть, но мне бы так больше понравилось без единички.

Daeberdir commented on code/modules/reagents/chemistry/reagents/medicine.dm:
```diff
@@ -1356,55 +1356,55 @@
+			if(prob(20))
+				to_chat(M, span_warning("Your skin feels hot and your veins are on fire!"))
 				update_flags |= M.adjustFireLoss(1, FALSE)
 			for(var/datum/reagent/R in M.reagents.reagent_list)
```
```suggestion
			if(has_stimulant)
				for(var/datum/reagent/R in M.reagents.reagent_list)
```
Тут прогонку по списку реагентов второй раз стоит пропустить, если в первый раз ничего интересного в них не нашлось.
Но там чуть ниже уже сам протабулируй.

Ksaikok46 committed:
corrections
